tests/test_generators/test_plantuml.py

```diff
@@ -16,14 +16,14 @@
 """
 
 PERSON = """
-class "Person" {
-    {field} id: string [req]
-    {field} name: string [opt]
-    {field} age_in_years: integer [opt]
-    {field} species_name: string [opt]
-    {field} stomach_count: integer [opt]
-    {field} is_living: LifeStatusEnum [opt]
-    {field} aliases: string [0..*]
+class "Person" [[{A person, living or dead}]] {
+    {field} "id" : "string" [req]
```

Comment:
AFAICT plantuml will render this with the quotes - I don't think we want this?

Reply:
I see what you mean. Quoting the class name I think is required but the "fields" are not desireable.
```plantuml
@startuml

class "Person A" [[{A person, living or dead}]] {
    {field} id : string a [req]
    {field} name : string [opt]
    {field} age in years : integer [opt]
    {field} "species_name" : "string" [opt]
    {field} "stomach_count" : "integer" [opt]
    {field} "is_living" : "LifeStatusEnum" [opt]
    {field} "aliases" : "string" [0..*]
}
@enduml
```

![image](https://github.com/linkml/linkml/assets/39277823/83212715-fb7c-4da8-9452-e8f2c0035a4b)
Given this "mix" of quote and not quote .. I can remove the "field quote" because PlantUML is able to handle field names with spaces in the names correctly.
Is this acceptable?

Reply:
Certain"fields" in the above image still have quotes around them? like "species_name", "stomach_count", etc.?

Also, I'm wondering if we want [req] and [opt]? that is syntax that we started using in the yuml generator and it stuck, i don't know if that is standard syntax anywhere else. Same for showing cardinality.

Do we want all of that information in the class diagram?

Reply:
I wanted to show the "differences" to be sure this is what people wanted and that the "spaces" are handled. So, making the quotes "all go away" is the goal.
On the "req vs opt" there is a bit of a dilemma. Since we are talking about a "Model" of fields, there is no "optional" notation in PlantUML but in some languages a "field" can be optional. This is _different_ than a "range" with zero or more "instances" (e.g. array , list, link, etc). I personally find it valuable to call this out given the limitations of PlantUML.
I am happy to make this work however everyone sees fit.
Quotes being removed is a "given".

Reply:
BTW, there are several threads on this "optional" problem in PlantUML in many forums.